deploy/osd-limited-support/limited-support.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: openshift-osd-metrics
```

Comment:
Does this namespace exist?

Reply:
Yes, this namespace is present in both Prod and Stage clusters.